PlaygroundComponent.jsx

```diff
@@ -8,47 +8,59 @@ import { playground_requests } from "./Playground_Requests";
 import SchemaTitle from "./components/Playground/SchemaTitle";
 import data_get_api_token from "./data-app-registration";
 import styles from "./PlaygroundComponent.module.scss";
-import { ticksSubject } from './ticksSubject';
+import { ticksSubject } from "./ticksSubject";
 import { createSignal } from "solid-js";
+import { useLocation } from "react-router-dom";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createBrowserHistory } from "history";
 
 export const PlaygroundComponent = () => {
-  const [current_api, setCurrentAPI] = useState(api)
-  const [is_initial_socket, setIsInitialSocket] = useState(true)
-  const [messages, setMessages] = useState([])
+  const [current_api, setCurrentAPI] = useState(api);
```

Comment:
since you are working on prettier changes, can you for now do not provide those changes here in this PR? I would like to see Prettier changes in seperate PR